src/client/driver/driver.js

```diff
@@ -371,24 +466,97 @@ export default class Driver {
             })
             .then(childDriverLink => {
                 childDriverLink.availabilityTimeout = commandSelectorTimeout;
-                this.activeChildDriverLink          = childDriverLink;
+                this.activeChildIframeDriverLink          = childDriverLink;
                 this.contextStorage.setItem(ACTIVE_IFRAME_SELECTOR, selector);
             });
     }
 
+    _waitForCommandCompletion () {
+        return waitFor(() => {
+            return !this.contextStorage.getItem(this.COMMAND_EXECUTING_FLAG);
+        }, COMMAND_EXECUTION_CHECK_DELAY, COMMAND_EXECUTION_MAX_TIMEOUT);
+    }
+
+    _switchToChildWindow (selector) {
```

Comment:
As far as I understand `selector` is pageId. Or it can be smth else? Can we rename this parameter? I mean that the `selector` word is confusing

Reply:
At present, the `selector` is only the pageId. 
Later it can be number or predicate.
```js
await t.switchToWindow (1);

await t.switchToWindow(wnd => wnd.title === 'Example page');
```

Reply:
May be `windowSelector` or `pageSelector` would be better. Or smth with `handler`. I do not insist

Reply:
In my opinion, the paremeter name `windowSelector` or `pageSelector` is a unnecessary context for function with name `switchToWindow`.

Similar with naming of `_switchToIframe (selector, iframeErrorCtors) {`. We don't use `elementSelector` name.

Reply:
But in `switchToIframe` selector is real Selector in terms of `testcafe`. But window selector is not Selector, it's number, or pageId, or predicate. I just want to prevent disambiguations.